rdbms/src/main/java/io/spine/server/storage/jdbc/JdbcAggregateStorage.java

```diff
@@ -48,6 +52,8 @@
 public class JdbcAggregateStorage<I> extends AggregateStorage<I> {
 
     private final DataSourceWrapper dataSource;
+    /** Iterators which are not closed yet. */
```

Comment:
Please explain yourselves in more details.

Reply:
Done.